app/models/claim.rb

```diff
@@ -136,27 +136,25 @@ class Claim < ApplicationRecord
 
   validates :payroll_gender, on: [:gender, :submit], presence: {message: "Choose the option for the gender your school’s payroll system associates with you"}
 
-  validates :first_name, on: [:name, :"personal-details", :submit], presence: {message: "Enter your first name"}
-  validates :first_name, length: {maximum: 100, message: "First name must be 100 characters or less"}, unless: :has_ecp_policy?
+  validates :first_name, on: [:"personal-details", :submit], presence: {message: "Enter your first name"}
   validates :first_name,
     on: [:"personal-details", :submit],
     length: {
       in: 2..30,
       message: "First name must be between 2 and 30 characters"
     },
-    if: -> { :has_ecp_policy? && first_name.present? }
+    if: -> { first_name.present? }
 
   validates :middle_name, length: {maximum: 100, message: "Middle name must be 100 characters or less"}
```

Comment:
@martikat Might want to check with Andy to see if we want consistency across all our names.